charts/CSVGenerator.tsx

```diff
@@ -108,18 +108,24 @@ export class CSVGenerator {
         dim: ChartDimensionWithOwidVariable,
         entity: string
     ): [string, string | number] | null {
-        let latestYear = undefined
-        let latestValue = undefined
-
-        const latestYearValue = dim.yearAndValueOfLatestValueforEntity(entity)
-
-        if (latestYearValue) {
-            latestYear = dim.formatYear(first(latestYearValue) as number)
-            latestValue = last(latestYearValue)
+        let year = undefined
+        let value = undefined
+
+        if (dim.targetYear && dim.tolerance) {
```

Comment:
`dim.tolerance` can be `0`, in which case it is falsy. Is this intentional? Shouldn't the targetYear be joined anyway, even if the tolerance is `0`?